src/component/notification.js

```diff
@@ -80,6 +80,33 @@ Alert.propTypes = {
   style: View.propTypes.style,
 };
 
+//
+// CountBubble
+//
+
+const countStyles = StyleSheet.create({
+  bubble: {
+    fontSize: font.sizeXS,
+    lineHeight: font.lineHeightXS,
+    paddingTop: 1,
+    paddingBottom: 2,
+    paddingRight: 10,
+    paddingLeft: 10,
+    borderRadius: 13,
+    backgroundColor: color.pinkSig,
+  },
+});
+
+export const CountBubble = ({ children, style }) =>
+  children && children !== '0' ? (
```

Comment:
In the past I've benefited from setting a `display: none` or `opacity: 0` property in these situations, to prevent the missing component from unexpectedly rearranging other components. but might not matter here

Reply:
That's a good point for elements that relevant for layout. In this case the count bubble should actually only render for non zero values.